app/src/sagas/transaction.js

```diff
@@ -47,7 +47,14 @@ function* sendTransaction({ transaction, title, type }) {
     yield put(addTx(txHash, title, type))
     yield put(dumpTxs())
   } catch (error) {
-    alert('Cannot send this transaction')
+    const currentUser = yield select(currentUserSelector)
```

Comment:
You should use checking condition for the error message if gas is insufficient and other error use alert.